data/json/items/generic.json

```
@@ -2401,6 +2401,21 @@
     "to_hit": -2,
     "flags": [ "TRADER_AVOID", "FIREWOOD" ]
   },
+  {
+    "type": "GENERIC",
```

```suggestion
    "type": "GENERIC",,
```
Making changes to rebase and reset tests